client/src/components/Signup/Signup.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Footer } from '../Footer/Footer';
+import './Signup.css'
+
+export default class Signup extends Component {
+  state = {
+    category: '',
+    first_name: '',
+    last_name: '',
+    email: '',
+    password: '',
+    password_repeat: ''
+  }
+  changeHandler = (e) => {
+    const key = e.target.name;
+    const val = e.target.value;
+    this.setState({ [key]: val })
+  }
+  changeSubmit = (e) => {
+    e.preventDefault();
+    //Validation check
+    if (!/^[a-zA-Z]+$/.test(this.state.first_name)) {
```

Comment:
Make a util folder and add a validation.js file and all the logic there so it can be reused with the other forms as well.